lib/config-generator.js

```diff
@@ -67,8 +68,13 @@ class ConfigGenerator {
 
         config.stats = this.buildStatsConfig();
 
+        let extensions = ['.js', '.jsx'];
+        if (this.webpackConfig.useTypeScriptLoader) {
+            extensions = extensions.concat(['.json', '.ts', '.tsx']);
+        }
```

Comment:
See my 2nd bullet point here: https://github.com/symfony/webpack-encore/pull/49#issuecomment-310548331. Perhaps we should add `.ts` and `.tsx` to extensions *always*.